### PR TITLE
[AMORO-2227] Fix displaying the change store metrics of Mixed KeyedTable

### LINKED ISSUE #2227: [Bug]: The table detail page does not display the change store metrics of Mixed KeyedTable
### What happened?

image

### Affects Versions

0.6.0

### What engines are you seeing the problem on?

AMS

### How to reproduce

Create a Mixed format Table with a primary key.

### Relevant log output

_No response_

### Anything else

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by wangtaohz**
## Why are the changes needed?

Close #2227 .

## Brief change log

- set change metrics for KeyedTable
- remove some useless codes

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (yes / **no**)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / **not documented**)

### CODE REVIEW

**Comment by wangtaohz** [comment]
image
image
image

**Comment by wangtaohz** [comment]
This should be cherry-picked to 0.6.x @shidayang

**Review by zhoujinsong — APPROVED** [review]
LGTM.

File: ams/server/src/main/java/com/netease/arctic/server/dashboard/MixedAndIcebergTableDescriptor.java [hunk 1]
```diff
@@ -127,24 +126,28 @@ public ServerTableMeta getTableDetail(AmoroTable<?> amoroTable) {
...
+        tableSize += changeFilesStatistics.getTotalSize();
+        tableFileCnt += changeFilesStatistics.getFileCnt();
+      } else {
+        changeMetrics.put("lastCommitTime", null);
+        changeMetrics.put("totalSize", null);
+        changeMetrics.put("fileCount", null);
+        changeMetrics.put("averageFileSize", null);
+        changeMetrics.put("tableWatermark", null);
+      }
+      serverTableMeta.setChangeMetrics(changeMetrics);
```
Comment (wangtaohz) [inline ams/server/src/main/java/com/netease/arctic/server/dashboard/MixedAndIcebergTableDescriptor.java:150]: This line is the main change. 😊

**Pull request merged**